symfony/templates/notice/edit.html.twig

```diff
@@ -32,7 +32,7 @@
                 </div>
                 <div class="panel-body">
                     {% set return = path('admin_notice', { notice: notice ? notice.id : null }) %}
-                    <p><strong>Your time zone</strong>: {{ app.userTimezoneDisplay }} <a href="{{ path('settings', { return: return}) }}" class="small">change</a></p>
+                    <p><strong>Your time zone</strong>: {{ app.user.info.timezone|default('Not set!') }} <a href="{{ path('settings', { return: settings_return_url }) }}" class="small">change</a></p>
```

Comment:
This displays `America/Chicago` instead of `Central Time`. To maintain one-for-one functionality, the lookup method either needs to be added/referenced to the `User` entity or elsewhere.